dhcpv6/nclient6/client.go

```diff
@@ -90,15 +90,26 @@ func NewIPv6UDPConn(iface string, port int) (net.PacketConn, error) {
 	})
 }
 
-// New creates a new DHCP client that sends and receives packets on the given
-// interface.
-func New(ifaceHWAddr net.HardwareAddr, opts ...ClientOpt) (*Client, error) {
+// New returns a new DHCPv6 client.
+func New(ifaceName string, ifaceHWAddr net.HardwareAddr, opts ...ClientOpt) (*Client, error) {
```

Comment:
A question that slipped through the cracks in the v4 PR - do we need to specify interface name and hardware address explicitly? We could get the hardware address from the interface name. Am I missing something?

Reply:
not missing anything, fixed. will post a v4 PR.